SonarQube.Common/AnalysisProperties/Property.cs

```diff
@@ -0,0 +1,99 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Property.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Xml.Serialization;
+
+namespace SonarQube.Common
+{
+    /// <summary>
+    /// Data class to describe an additional analysis configuration property
+    /// /// </summary>
+    /// <remarks>The class is XML-serializable</remarks>
+    public class Property
```

Comment:
Very similar to AnalysisSetting, but the XML serialization format is slightly different because of the attributes on the properties.